src/Services/UserRegisterService.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Services;
+
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Auth\Events\Registered;
+use Illuminate\Contracts\Hashing\Hasher;
+
+class UserRegisterService
+{
+    public function __construct(
+        protected Hasher $hasher,
+    ) {}
+
+    public function register(string $email, string $name, string $password): void
+    {
+        if (!$this->hasher->info($password)["algo"]) {
```

Comment:
I think that we don't need to check this condition anymore.

Reply:
done